web/src/routes/OrganizationsList/Organization/Tabs/TeamsAndMembership/TeamsView/ManageMembers/ManageMembersList.tsx

```diff
@@ -333,109 +411,255 @@ export default function ManageMembersList(props: ManageMembersListProps) {
     return <Spinner />;
   }
 
+  const removeTeamSyncModalHolder = (
+    <ConfirmationModal
+      title="Remove Synchronization"
+      description="Are you sure you want to disable group syncing on this team? The team will once again become editable."
+      buttonText="Confirm"
+      modalOpen={isRemoveTeamSyncModalOpen}
+      toggleModal={() => setRemoveTeamSyncModalOpen(!isRemoveTeamSyncModalOpen)}
+      handleModalConfirm={() => removeTeamSync()}
+    />
+  );
+
+  const displaySyncDirectory =
+    teamCanSync != null &&
+    !teamSyncInfo &&
+    config?.registry_state !== 'readonly';
+
+  const fetchSyncBtn = () => {
+    const result = [];
+    if (displaySyncDirectory) {
+      result.push(
+        <Button
+          variant="link"
+          onClick={() => setIsOIDCTeamSyncModalOpen(!isOIDCTeamSyncModalOpen)}
+          key="team-sync-btn"
+        >
+          Enable Directory Sync
+        </Button>,
+      );
+    }
+    if (pageInReadOnlyMode) {
+      result.push(
+        <Button
+          onClick={() => setRemoveTeamSyncModalOpen(!isRemoveTeamSyncModalOpen)}
+          key="remove-team-sync-btn"
+        >
+          Remove synchronization
+        </Button>,
+      );
+    }
+    return result;
+  };
+
+  const teamSyncedConfig = (
+    <>
+      <Alert
+        isInline
+        variant="info"
+        title={`This team is synchronized with a group in ${teamCanSync?.service} and its user membership is therefore read-only.`}
+        id="teamsync-readonly-alert"
+      />
+      <Conditional if={OIDCGroupName != null}>
+        <Accordion>
+          <AccordionItem>
+            <AccordionToggle
+              onClick={() => {
+                onAccordionToggle('oidc-config-toggle');
+              }}
+              isExpanded={teamSyncConfigExpanded === 'oidc-config-toggle'}
+              id="oidc-config-toggle"
+            >
+              Directory Synchronization Config
+            </AccordionToggle>
+            <AccordionContent
+              id="oidc-config-toggle"
+              isHidden={teamSyncConfigExpanded !== 'oidc-config-toggle'}
+            >
+              <TextContent>
+                <TextList component={TextListVariants.dl}>
+                  <TextListItem component={TextListItemVariants.dt}>
+                    Bound to group
+                  </TextListItem>
+                  <TextListItem component={TextListItemVariants.dd}>
+                    {OIDCGroupName}
+                  </TextListItem>
+                  <TextListItem component={TextListItemVariants.dt}>
+                    Last Updated
+                  </TextListItem>
+                  <TextListItem component={TextListItemVariants.dd}>
+                    Never
```

Comment:
Hardcoded value, i'm assuming this will be updated in the future?

Reply:
Good catch, I missed this. Updated to read from state.